samples/readme.samples.md

```diff
@@ -4,4 +4,20 @@
 require: $(this-folder)/../readme.md
 output-folder: $(this-folder)/$(title)/$(title)
 clear-output-folder: false
+```
+
+``` yaml $(include-csproj) != 'disable'
```

Comment:
For whatever reason, boolean values don't work for these conditions. However, string comparisons work! So, we use 'enable/disable' for this stuff. By default, we make a csproj and build. However, for Azure.Storage.Tables, we disable generating the csproj.

Reply:
Can we use a `true` as a string? Should we file this issue?

Reply:
I did 'enable/disable' to avoid any confusion with booleans. And, I thought there was an issue filed before... when we ran into this like a few months back. But, I could be wrong.